### ISSUE TITLE
If there are more than 6 menus tabs on the top level menu. I am unable to see remaining menu tabs towards right.

### ISSUE THREAD

**Issue by kiransk9** [google/docsy#1058]
Hi, 

I am pretty new to Hugo and Docsy. But enjoying learning and building cool static website.

We have almost 9 menu tabs on the top level menu bar. When I am trying to access the website in 1280*1024 resolution and 100% zoom . I am unable to see the remaining menu tabs toward right.. Is there any option we can wrap down the tabs when accessing in lower resolution.

Thank you in advance.

SK

**Comment by fekete-robert** [comment]
You're right, also the title of the site isn't wrapped when it's longer. Seems like the top header isn't as responsive as the rest of the site.

**Comment by kiransk9** [comment]
I will try to debug and see if I can write some snippet to achieve that .. But, if already any implementation please let me know.

Thanks

**Comment by LisaFC** [comment]
The top header currently scrolls across when there are more menu items than fit on the screen but it's not always obvious that it's scrollable. Going to think about how best to signal that without a scrollbar.  The other alternative is, as you say, wrapping down the menu items but that

[Kubeflow](https://www.kubeflow.org/docs/) (a Docsy site with some customization) have tweaked their navbar so the whole thing is a drop-down on mobile - would that work?

Ideas and examples welcome!

**Comment by kimberleybrown** [comment]
A hamburger menu with dropdown would be a great addition I think for the mobile version. Is there a possibility that we could make this a config toggle in case some folks prefer what's originally there for mobile?

**Comment by LisaFC** [comment]
(sorry, was on holidays so just catching up on messages)

I'll have a look!
